kai-analyzer/provider/java/internal_java_provider.go

```diff
@@ -0,0 +1,44 @@
+package java
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	"github.com/konveyor/analyzer-lsp/provider"
+	extjava "github.com/shawn-hurley/java-external-provider/pkg/java_external_provider"
```

Comment:
Is there something special in your provider?

```suggestion
	extjava "github.com/konveyor/analyzer-lsp/external-providers/java-external-provider"
```

Reply:
I fixed a bug, that emily fixed, need to update